tests/summ_ai_api/summ_ai_test.py

```diff
@@ -228,3 +437,186 @@ async def test_summ_ai_error_handling(
             "Please try again later or contact an administrator",
             response,
         )
+
+
+def test_validate_response_valid():
+    """
+    Test for validate_response method.
+    Should return True when 'translated_text' is found in request.
+    """
+    response_data_translation_succeeded = {
+        "dummy": "dummy",
+        "translated_text": "Das ist einfaches Deutsch.",
+    }
+    assert (
+        SummAiApiClient(MockedRequest(), PageTranslationForm).validate_response(
+            response_data_translation_succeeded, 200
+        )
+        is True
+    ), "if translated_text found in reponse-data, validate_response should return True"
+
+
+def test_validate_response_invalid():
+    """
+    Test for validate_response method.
+    Should throw exception when 'translated_text' is not found in request.
+    """
+    response_data_translation_failed = {
+        "dummy": "dummy",
+        "error": "no translation test error",
+    }
+    with pytest.raises(SummAiRuntimeError):
+        SummAiApiClient(MockedRequest(), PageTranslationForm).validate_response(
+            response_data_translation_failed, 200
+        )
+
+
+async def test_invalid_json(settings, aiohttp_raw_server):
+    """
+    Test correct handling for invalid json response by SUMM.AI.
+    Tests if SummAiInvalidJSONError is raised when an invalid JSON is sent as response.
+    """
+    session = aiohttp.ClientSession()
+    fake_server = await aiohttp_raw_server(
+        make_broken_json_summ_ai_server(
+            response_text="""{
+                "translated_text": "a text in an unfinished json response",
+                "jobid": "9999",
+            """,
+            status=200,
```

Comment:
```suggestion
            response_text="<h1>Internal Server Error</h1>",
            status=500,
```
A more realistic example wold be an html-response...

Reply:
I think there would be 2 separate cases:
1. Unexpected html-response with 200 status code
2. 500 status code

Or even 3:
3. 200 status code, json response without translated_text tag